djangomodelimport/forms.py

```diff
@@ -1,92 +1,15 @@
 from django import forms
 
-from .fields import CachedChoiceField, FlatRelatedField
-from .loaders import CachedInstanceLoader
+from .magic import CachedChoiceFieldFormMixin, FlatRelatedFieldFormMixin, JSONFieldFormMixin
 
 
-class ImporterModelForm(forms.ModelForm):
+class ImporterModelForm(JSONFieldFormMixin, FlatRelatedFieldFormMixin, CachedChoiceFieldFormMixin, forms.ModelForm):
```

Comment:
Mixins don't seem ideal here. I think it'd make sense if we were just able to register these other classes (eg with a decorator) such that they magically mixed themselves in, or responded to signals that this class sent off.